server/src/main/java/org/elasticsearch/index/mapper/NestedValueFetcher.java

```diff
@@ -28,6 +29,7 @@
     // the name of the nested field without the full path, i.e. in foo.bar.baz it would be baz
     private final String nestedFieldName;
     private final String[] nestedPathParts;
+    private LeafReaderContext currentContext;
```

Comment:
I don't think we need to track the context anymore.

Reply:
sure, leftover. I pushed an update